src/agent/src/unix_main.cpp

```diff
@@ -21,6 +22,15 @@ int main(int argc, char* argv[])
         }
         else if (cmdParser.OptionExists("start"))
         {
+            int ret = AgentDaemon::Daemonize();
```

Comment:
Can be const!

Reply:
Done. This code was moved to process_options_unix.cpp